### PR TITLE
xarray_mongodb recipe

### PR DESCRIPTION
**Pull request by crusaderky**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by crusaderky** [comment]
@conda-forge/help-python ready for review and merge

File: recipes/xarray_mongodb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  home: https://github.com/AmphoraInc/xarray_mongodb
+  license: APACHE-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: Store xarray objects in MongoDB
+
+extra:
+  recipe-maintainers:
+    - crusaderky
+    - gimperiale
```
Comment (scopatz) [inline recipes/xarray_mongodb/meta.yaml]: @gimperiale are you willing to be a maintainer?

Comment (gimperiale) [reply recipes/xarray_mongodb/meta.yaml]: We're the same person. This is the corporate account whereas @crusaderky is the personal one.

Comment (scopatz) [reply recipes/xarray_mongodb/meta.yaml]: haha ok

**Pull request merged**